### ISSUE TITLE
Fails Semver Check

### ISSUE THREAD

**Issue by coreybutler** [acornjs/acorn-private-class-elements#7]
Due to an [invalid semver check](https://github.com/acornjs/acorn-private-class-elements/blob/master/index.js#L4-L6), this entire library fails in Acorn 7+.

![image](https://user-images.githubusercontent.com/770982/71144604-35608700-21e4-11ea-8a93-3ec31fbecede.png)

**Comment by Katherraptor** [comment]
Bump

**Comment by calebdwilliams** [comment]
Trying to use this plugin with Rollup and getting the same error. Is there any timeline for a fix?

**Comment by coreybutler** [comment]
In case it helps anyone else, I was originally attempting to use this with Rollup too. I was trying to avoid Babel, but never found a workaround. So, I begrudgingly ended up using the Babel plugin with a config like this:

```javascript
babel({
    presets: [['@babel/preset-env', { targets: { node: true } }]],
    plugins: [
      ['@babel/plugin-proposal-class-properties', { loose: false }],
      ['@babel/plugin-proposal-private-methods', { loose: false }]
    ],
    externalHelpersWhitelist: ['classPrivateFieldSet', 'classPrivateFieldGet', 'classPrivateMethods']
  })
```

**Comment by adrianheine** [comment]
I've just published 0.2.0 with #2 as merged by marijn.
